doc/source/installation.rst

```diff
@@ -10,16 +10,40 @@ Environment
 Installation
 ============
 
+Since there is no stable release yet, the only option is to install the project from the source.
 
-1. Install http://github.com/tendrl/bridge_common
-2. Install http://github.com/tendrl/gluster_bridge
+Development version from the source
+-----------------------------------
 
-    At the command line::
+1. Install http://github.com/tendrl/bridge_common from the source code::
+
+    $ git clone https://github.com/Tendrl/bridge_common.git
+    $ cd bridge_common
+    $ mkvirtualenv bridge_common
+    $ pip install .
```

Comment:
Earlier it was like that. But better user refers one and has the setup ready. so its OK I feel. Still if everybody feels strongly to refer from common, can be done :)